static/common/js/registration.js

```diff
@@ -0,0 +1,47 @@
+getById = id => document.getElementById(id);
+
+
+let nick = getById("nickname");
```

Comment:
IMHO this would be more usable/readable if you'd add this to an object: 
```
let formData = {
"nick": getById("nickname"),
(...)
}
```

so you can later `formData.nick`, but it would also allow you to iterate over the fields with for cycle. 

YMMV, treat it as an opinion ;)